packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts

```diff
@@ -63,22 +64,24 @@ export class ModalNotification extends AbstractDialog<string | undefined> {
 
         const textContainer = messageNode.appendChild(document.createElement('div'));
         textContainer.classList.add(TEXT);
-        const textElement = textContainer.appendChild(document.createElement('span'));
+        const textElement = textContainer.appendChild(document.createElement('pre'));
         textElement.textContent = text;
 
-        actions.forEach((action: string) => {
-            const button = this.createButton(action);
+        actions.forEach((action: MainMessageItem) => {
+            const button = this.createButton(action.title);
             button.classList.add('main');
             this.controlPanel.appendChild(button);
             this.addKeyListener(button,
                 Key.ENTER,
                 () => {
-                    this.actionTitle = action;
+                    this.actionTitle = action.title;
                     this.accept();
                 },
                 'click');
         });
-        this.appendCloseButton('close');
+        if (!actions.some(action => action.isCloseAffordance === true)) {
+            this.appendCloseButton('close');
```

Comment:
nit: in most cases first upper case is used. this isn't changed here, I just saw this while testing

Reply:
> nit: in most cases first upper case is used. this isn't changed here, I just saw this while testing

@AlexTugarev do you mind clarifying? (what sort of change is required)

Reply:
Ping :)

Reply:
missed that ;-)
it was not important. `Close` would be better aligned with other button titles. nvm.

Reply:
No problem, thank you for the review! 😄